sycl/test-e2e/lit.cfg.py

```diff
@@ -413,7 +413,7 @@
             config.available_features.add("gpu-amd-gfx90a")
         if not line.startswith("["):
             continue
-        (backend, device, _) = line[1:].split(":", 2)
+        (backend, device) = line[1:].split("]")[0].split(":")
```

Comment:
I understand this change is anything to prevent any e2e test from breaking. Do we need something similar for `sycl/test`?

Reply:
We don't call `sycl-ls` in `sycl/test/lit.cfg.py`, so same change there isn't needed/possible.